### DISCUSSION TITLE
1.1.14: Different version in Github and from HACS

### DISCUSSION THREAD

**Discussion by salosa** [veista/nilan#48] [General]
I'm not sure, but if I get latest version of this Nilan integration 1.1.14 from HACS. I received old version of some files. Is it possible?
e.g sensor.py
 "get_display_text_1": [
        Map(
            "Display Text Line 1",
            None,
            None,
            SensorStateClass.MEASUREMENT,
            None,
            None,
            None,
        )
    ],
    "get_display_text_2": [
        Map(
            "Display Text Line 2",
            None,
            None,
            SensorStateClass.MEASUREMENT,
            None,
            None,

**Comment by veista** [comment]
Not quite sure I understand the question. HACS shows only published releases, while you can see all development branches in guthub.

**Reply by salosa** [reply]
My mistake. I thought that those fixes are already in 1.1.14 release. Sorry
